.github/ISSUE_TEMPLATE/meeting_setup.md

```diff
@@ -18,24 +20,30 @@ Steps to take when setting up a new meeting...
 + [ ] **Request details from speaker 2** : Request a summary of the talk and a speaker biography from speaker 2.
 + [ ] **Create Calendar Event** : Create a calendar event in Google Calendar. (Calendar Link can be found under _More
       Actions > Publish  Event_).
-      + [ ] Google Meet Settings : Meeting access type set to Open
-      + [ ] Google Meet Link :
-      + [ ] Calendar Link :
+  + [ ] Google Meet Settings : Meeting access type set to Open
+  + [ ] Google Meet Link :
+  + [ ] Calendar Link :
 + [ ] **Website Post** : Create a new post (see [README](https://github.com/SheffieldR/SheffieldR.github.io) for
       instructions) with details of the talk(s) and speaker(s) biographies. Include both the Google Meet and Calendar
       Link in the web-page.
-+ [ ] **Announce on Meetup Mailing List** : Announce the meeting on the [Meetup SheffieldR User
+
+## Announcements
+
+Announcements of forthcoming events can be made in a number of places, at a bare minimum the _Meetup Mailing List_
+should be used as it is the primary source for organising the group.
+
++ [ ] **Meetup Mailing List** : Announce the meeting on the [Meetup SheffieldR User
       Group](https://www.meetup.com/sheffieldr-sheffield-r-users-group/) mailing list including both the Google Meet and
       Calendar links.
-+ [ ] **Announce on RSE Mailing List** : Announce the meeting on the [RSE Sheffield Mailing
++ [ ] **RSE Mailing List** : Announce the meeting on the [RSE Sheffield Mailing
       List](https://groups.google.com/a/sheffield.ac.uk/g/RSE-group)) mailing list including both the Google Meet and
       Calendar links.
-+ [ ] **Announce on NHS-R Slack** : If you are a member you can announce the event on the [NHS-R
-      Slack](https://nhsrcommunity.slack.com/).
-+ [ ] **Announce on Sheffield Digital Slack** : If you are a member you can announce the event on the [Sheffield Digital
-      Slack](https://sheffielddigital.slack.com).
++ [ ] **University Website - WhatsOn** : Submit [a form]( https://www.sheffield.ac.uk/whatson/submit ) with details of
```

Comment:
Might be good to add something about how long is needed to get an event listed if we know? (Istr somebody mentioning this but I can't remember what they said.)

Reply:
I recall reading this somewhere myself but can't find it now. 

I've put advice in the paragraph at the start of this section that it should be done two weeks in advance to maximimise notification for people to attend.